src/locales/en.json

```diff
@@ -199,6 +199,7 @@
   },
   "next": "Next",
   "no-events-on-selected-day-message": "There are no events on selected day",
+  "no-events-message-all": "There are no events",
```

Comment:
Please sort properties alphabetically (action available in WebStorm quick action menu)